raiden/network/transport/matrix/client.py

```diff
@@ -258,6 +258,28 @@ def __init__(
         self._presence_update_ids: Iterator[int] = itertools.count()
         self._worker_pool = gevent.pool.Pool(size=20)
 
+        self._sync_filter_id: Optional[int] = None
+
+    def create_sync_filter(self, broadcast_rooms: Dict[str, Optional[Room]]) -> Optional[int]:
+        ignore_rooms = [room.room_id for room in broadcast_rooms.values() if room is not None]
+        filter_params = {
+            "room": {
+                # Filter out all room state events / messages from broadcast rooms
+                "not_rooms": ignore_rooms,
```

Comment:
https://matrix.org/docs/spec/client_server/latest#post-matrix-client-r0-user-userid-filter

This simplifies things as instead of specifying the types / rooms we want to include/exclude, we just filter out anything from the ignored rooms (state events & messages)

> rooms | [string] | A list of room IDs to include. If this list is absent then all rooms are included. This filter is applied before the filters in ephemeral, state, timeline or account_data